### PR TITLE
Use same font as website for the logo

### PR DESCRIPTION
**Pull request by amousset**
How would you feel about using the same font for the logo as on https://www.darktable.org/ ? I feel like it could help maintaining a consistent appearance and look a bit more polished.

exo
(only a quick-and-dirty draft for now)

### CODE REVIEW

**Comment by Nilvus** [comment]
Visually, why not. But this will need that all users have font installed. And that's a bad thing to add such specific font for that reason. Different users will have different rendering depending on fonts installed on their system. So not safe.

**Comment by AlicVB** [comment]
one solution could be to use an image for 'darktable', to avoid any font problems. (to be checked if text color stay the same in all themes)

**Comment by amousset** [comment]
This approach is indeed not practical, but it could probably be an svg with the text converted to paths.

**Comment by Nilvus** [comment]
An svg would indeed be better.

**Comment by TurboGit** [comment]
Agreed an SVG would better !

File: src/common/utility.c [hunk 1]
```diff
@@ -377,17 +377,11 @@ dt_logo_season_t dt_util_get_logo_season(void)
   return DT_LOGO_SEASON_NONE;
 }
 
-cairo_surface_t *dt_util_get_logo(const float size)
+static cairo_surface_t *dt_util_get_svg_img(gchar *logo, const float size)
```
Comment (TurboGit) [inline src/common/utility.c:380]: static routine should not start with dt_ but just _ (a coding convention).

**Review by TurboGit — CHANGES_REQUESTED** [review]
A minor comment on style apart from that it looks good and work fine.

File: src/common/utility.c (hunk 1 above)
Comment (amousset) [reply src/common/utility.c]: :+1:

**Comment by TurboGit** [comment]
All fine now, and thanks for this nice little detail for darktable visual identity !

**Pull request merged**